app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -122,5 +119,28 @@ def process_errata_ids(select_all_params)
         []
       end
     end
+
+    private def create_rules
```

Comment:
@akofink if only there was a project that allowed us to establish and enforce syntax rules ;)